### DISCUSSION TITLE
Can't show dock on multiple monitors

### DISCUSSION THREAD

**Discussion by Semperverus** [linuxdeepin/developer-center#2949] [Features Request & Ideas | 特性请求 & 头脑风暴]
I'm running a triple monitor setup, but I can't seem to get the dock to show on anything but the primary display. Ideally, it would duplicate itself to the other monitors as well.

In relation to this, it appears as though you can't right-click the desktop on any monitor except the primary one as well.

**Comment by ghost** [comment]
deepin is not developed for multi monitor
you will only see the wallpaper

**Comment by BLumia** [comment]
I guess you posted to the wrong repo, the dock project is [dde-dock](https://github.com/linuxdeepin/dde-dock) and deepin use [developer-center](https://github.com/linuxdeepin/developer-center) for general feedback.

cc @xmuli

**Comment by XMuli** [comment]
HI~, I only found one dual screen, the picture below can be set to change the main screen, the dock can also be changed to another screen, you can try it. Hope that helps.

![image](https://user-images.githubusercontent.com/33887845/82513338-4d4a8700-9b45-11ea-8db6-19e803c0729a.png)

**Comment by XMuli** [comment]
> I'm running a triple monitor setup, but I can't seem to get the dock to show on anything but the primary display. Ideally, it would duplicate itself to the other monitors as well.

I've asked later, and it's one of those that doesn't support this feature yet, so please expect it.

**Comment by scroom** [comment]
> > I'm running a triple monitor setup, but I can't seem to get the dock to show on anything but the primary display. Ideally, it would duplicate itself to the other monitors as well.
> 
> I've asked later, and it's one of those that doesn't support this feature yet, so please expect it.

Does this mean we can expect such a feature in a future release?

**Comment by 0yanghongwei** [comment]
Hi. This question is very similar to the following one. The following one has been resolved, you can see if it meets your requirements.
[Dock goes to the secondary monitor in dual monitors linuxdeepin/dde-dock#351](https://github.com/linuxdeepin/dde-dock/issues/351)

**Comment by meiyixiang** [comment]
Version 20.5 provides two modes: one is to display the DOCK only on the main screen; the other is to follow the mouse. When you move the mouse to the bottom, the DOCK will be changed. Please upgrade the system
